docs/README.md

```diff
@@ -54,6 +54,34 @@ By default implementation we mean that whenever you use `checkout-summary` in yo
 
 Therefore, in order to customize the Checkout Summary configuration, you can simply use the default implementation in your code and change it as you wish.
 
+Additionally, you can also include a `"drawer"` block inside the `checkout-summary`, and the drawer trigger
+will appear alongside the "Summary" title.
```

Comment:
```suggestion
Additionally, you can also declare a [Drawer](https://vtex.io/docs/components/content-blocks/vtex.store-drawer@0.10.0) as children of `checkout-summary` block. 

Notice the following: the block declared as children of the `drawer-trigger` block will appear alongside the Summary title.
```

Reply:
> as children of `checkout-summary` block.

@guerreirobeatriz about the sentence above, the drawer actually goes in the `blocks` of `checkout-summary`, do you think I can write the following?

> you can also declare a [Drawer](https://vtex.io/docs/components/content-blocks/vtex.store-drawer) as a block of `checkout-summary`.